kickme.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+docker-compose down && docker-compose pull --ignore-pull-failures && docker-compose up -d
```

Comment:
FYI, this can be improved to minimize down time.

```diff
- docker-compose down && docker-compose pull --ignore-pull-failures && docker-compose up -d
+ docker-compose pull && docker-compose up -d --force-recreate
```

This will pull down the new image without stopping the currently running container, then recreate the container with the new image.

Reply:
Oops, yeah that's much better. I should use that for more of my things

How about the compose file? I feel like that's sort of useful for this repo at least. Would prevent other chumps like myself from needing to re-write it.

Reply:
The git repo is for building the Docker image and I believe most people pull the image directly, and don't clone the git repo. Also, the options in docker-compose (i.e. timezone and volumes) are subjective and would possibly differ for each individual setup.

Maybe we could put an example compose file in the `README`.

Reply:
Ah yeah. I think the timezone stuff could be replaced with:

`volumes:
    - "/etc/timezone:/etc/timezone:ro"
    - "/etc/localtime:/etc/localtime:ro"
`

to make it universal. I've used that for some of my other setups.

An example compose in the README might be nice. I think a lot of docker newcomers aren't necessarily technical enough to know about compose or be able to write their own configs. A short example and brief on compose could be good for the README